app/templates/minor/profile.html

```diff
@@ -1,92 +1,561 @@
+                        <label for="otherExperience">Other. Briefly describe your summer experience below if it will not be one of the other experiences listed.</label><br>
+                        <textarea class="form-control" rows="2" cols="72" id="otherExperienceDescription" name="otherExperienceDescription" {% if summer_experience.experienceType in ['Internship', 'Community-based research', 'Community-Based Labor Position', 'EPG Summer Internship', 'Bonner Scholar Summer of Service'] %}style="display:none;"{% endif %}>{{ summer_experience.experienceType if summer_experience.experienceType not in ['Internship', 'Community-based research', 'Community-Based Labor Position', 'EPG Summer Internship', 'Bonner Scholar Summer of Service'] else '' }}</textarea><br>
+                    </div>
+        
+                    <div class="form-group">
+                        <label><strong>Which of the four key content areas of the Community and Civic Engagement Minor will your Community-Engaged Summer Experience address? Please select all that apply.</strong></label><br>
+                        <input type="checkbox" id="powerInequality" name="contentArea" value="Power and inequality" {% if 'Power and inequality' in summer_experience.CceMinorContentArea %}checked{% endif %}>
+                        <label for="powerInequality">Power and inequality</label><br>
+                        <input type="checkbox" id="communityIdentity" name="contentArea" value="Community and identity" {% if 'Community and identity' in summer_experience.CceMinorContentArea %}checked{% endif %}>
+                        <label for="communityIdentity">Community and identity</label><br>
+                        <input type="checkbox" id="civicLiteracy" name="contentArea" value="Civic literacy" {% if 'Civic literacy' in summer_experience.CceMinorContentArea %}checked{% endif %}>
+                        <label for="civicLiteracy">Civic literacy</label><br>
+                        <input type="checkbox" id="civicSkills" name="contentArea" value="Civic skills" {% if 'Civic skills' in summer_experience.CceMinorContentArea %}checked{% endif %}>
+                        <label for="civicSkills">Civic skills</label><br><br>
+                    </div>
+        
+                    <div class="form-group">
+                        <label for="experienceHoursOver300"><strong>Will your experience include at least 300 hours of work over the span of at least 8 weeks?</strong></label><br>
+                        <input type="radio" id="yes300hours" name="experienceHoursOver300" value="Yes" {% if summer_experience.experienceHoursOver300 %}checked{% endif %} onclick="toggleTextarea()">
```

Comment:
We have already added a listener to these inputs, so we don't need to put an `onclick` in the html